bot/cogs/cyber.py

```diff
@@ -368,6 +370,17 @@ def suffix_from_number(num):
             text += "We’re currently allocating students to their preferred locations so it’s an ongoing process!"
             text += " We’ll send out details of your location as soon as we can. It shouldn’t be too long!"
             await message.channel.send(text)
+        # Level Function Regex.
```

Comment:
My personal preference would be to have an extra blank line before this. Your choice here.

```suggestion

        # Level Function Regex.
```